api/config/config_perf.json

```diff
@@ -0,0 +1,73 @@
+{
+  "firecloud": {
+    "baseUrl": "https:\/\/firecloud-orchestration.dsde-perf.broadinstitute.org",
+    "billingAccountId": "014D91-FCB792-33D2C0",
+    "billingProjectPrefix": "aou-rw-perf-",
+    "clusterMaxAgeDays": 7,
+    "clusterIdleMaxAgeDays": 3,
+    "debugEndpoints": false,
+    "enforceRegistered": true,
+    "leoBaseUrl": "https:\/\/leonardo.dsde-perf.broadinstitute.org",
+    "registeredDomainName": "all-of-us-registered-perf",
+    "registeredDomainGroup": "all-of-us-registered-perf@perf.test.firecloud.org",
+    "billingRetryCount": 4,
+    "billingProjectBufferCapacity": 20,
+    "xAppIdValue": "perf-AoU-RW",
+    "vpcServicePerimeterName": "accessPolicies/357236995176/servicePerimeters/terra_prod_aou_perf"
```

Comment:
Does this exist? Also is "terra_prod_" the correct prefix here?

Reply:
This does not exist. Current plan was to punt on VPC perimeter for perf for the time being. I'll change to TODO or something similar.